### PR TITLE
Scrutinizer improvements

### PR DESCRIPTION
**Pull request by lgnap**

### CODE REVIEW

File: src/Bpost/Order/Box.php [hunk 1]
```diff
@@ -264,7 +264,6 @@ public static function createFromXML(\SimpleXMLElement $xml)
             $className = '\\Bpost\\BpostApiClient\\Bpost\\Order\\Box\\' . ucfirst($internationalBoxData->getName());
 
             if (!method_exists($className, 'createFromXML')) {
-                var_dump($className);
```
Comment (kouinkouin) [inline src/Bpost/Order/Box.php:313]: We cannot remove the `var_dump` as is.  
If we remove the `var_dump`, we have to set a message in the `BpostNotImplementedException` to show which `$className` does not contain method `createFromXML`.

**Review by kouinkouin — APPROVED** [review]

File: src/Bpost/Order/Box.php (hunk 1 above)
Comment (lgnap) [reply src/Bpost/Order/Box.php:313]: I followed the pattern used at a a lot of place into code. But in fact seems missing. Added (and at others places also)

**Comment by lgnap** [comment]
@rpommier-antidot Tu la merges ou je la close?

**Comment by kouinkouin** [comment]
@lgnap https://github.com/Antidot-be/bpost-api-library/pull/13/files/05effa97a8acbc0ea412af399a9003c48e781448..eacfeb0b07af392d5d5d677f5699bab1c819d7e0#diff-f9b101a262c15825fa48fca6b231e139 ? :-)

**Pull request merged**